### PR TITLE
feat(config): add NAS-PD07ZU1

### LINKED ISSUE #3646: Missing device configuration: Haozee 5-in-1 Multisensor
### Checklist

- [X] It is not in the [configuration DB](https://devices.zwave-js.io/?search=3)

- [X] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)

### Which device is missing?

Haozee 5-in-1 Multisensor

### Manufacturer ID

0x0258

### Product Type

0x0020

### Product ID

0x0718

### Firmware Version

2.23

### Is the device listed on the Z-Wave Alliance website?

No

### Do you have a manual?
https://usermanual.wiki/NEO-ELECTRONICS/NAS-WR12Z1U-5245649.pdf

This is exactly the exact same device as:

https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x0258/nas-pd07z.json

https://devices.zwave-js.io/?jumpTo=0x0258:0x0010:0x0720:0.0

It is just a rebranded/relabelled version, and needs the type/id added so the correct driver is used.

### PR DESCRIPTION
**Pull request by Botched1**
Add new device configuration file for NAS-PD07ZU1

fixes: #3646

### CODE REVIEW

**Comment by Botched1** [comment]
Should be linked to:
https://github.com/zwave-js/node-zwave-js/issues/3646

File: packages/config/config/devices/0x0258/nas-pd07u1.json [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+			"defaultValue": 100,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 100
+				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-pd07u1.json]:
```suggestion
				{
					"label": "On (255)",
					"value": 100
				}
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Just a detail, the rest looks okay. Needs a final re-formatting after you're done.

File: packages/config/config/devices/0x0258/nas-pd07u1.json (hunk 1 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0258/nas-pd07u1.json]: The values were referring to the command that gets sent, not turning the feature on/off.

**Review by Botched1 — COMMENTED** [review]
Good change.

**Comment by Botched1** [comment]
> @zwave-js-bot fix lint

@zwave-js-bot fix lint

**Review by Botched1 — COMMENTED** [review]
OK

**Review by Botched1 — COMMENTED** [review]
OK

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**